README.md

```diff
@@ -36,8 +36,19 @@ Work needed includes:
 | Platform neutrality   |               |                                                  |
 
 ### Logistics
-- Monthly Meetings
-- Biannual F2F
+
+#### Semi-monthly Meetings
+
+Meetings of the working group typically occur every other Wednesday. A few days before each
+meeting, an [https://github.com/nodejs/diagnostics/issues](issue) will be created with the
```

Comment:
This should be `[issue](https://github.com/nodejs/diagnostics/issues)`.

Reply:
Ugh - not paying attention. Addressed.